en/dev_log/ulog_file_format.md

```diff
@@ -41,233 +48,318 @@ The file consists of three sections:
 ----------------------
 ```
 
+A description of each section is provided below.
+
+
 ### Header Section
 
 The header is a fixed-size section and has the following format (16 bytes):
+
 ```
 ----------------------------------------------------------------------
 | 0x55 0x4c 0x6f 0x67 0x01 0x12 0x35 | 0x01         | uint64_t       |
 | File magic (7B)                    | Version (1B) | Timestamp (8B) |
 ----------------------------------------------------------------------
 ```
 
-Version is the file format version, currently 1.
-Timestamp is a `uint64_t` integer, denotes the start of the logging in microseconds.
+- **File Magic (7 Bytes):** File type indicator that reads "ULog"
+- **Version (1 Byte):** File format version (currently 1)
+- **Timestamp (8 Bytes):** `uint64_t` integer that denotes when the logging started in microseconds.
 
+### Definition & Data Section Message Header
 
-### Definitions Section
+The _Definitions and Data_ section consists of a stream of **messages** that always starts with this header: 
```

Comment:
Are you saying that:
- every message in both sections starts with this header
- before all the stream of messages for the definitions and data this header is displayed
- before the stream of messages for the definitions section this header is display and also it is displayed again before the stream for data?

This sentence will need to change based on which one of the above is true.

Reply:
> * every message in both sections starts with this header

This is the correct case. I think I can change it to:

The _Definitions and Data_ section consists of **messages**, and each message always starts with this header:

Reply:
Do you mean "starts with" or is "preceded by":
```suggestion
The _Definitions and Data_ sections contain a number of **messages**. Each message is preceded by this header: 
```